src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/api/summaries/HotResourceSummary.java

```diff
@@ -124,17 +125,18 @@ public static HotResourceSummary buildHotResourceSummaryFromMessage(
     newSummary
         .setValueDistribution(message.getMinValue(), message.getMaxValue(), message.getAvgValue());
     if (message.hasConsumers() && message.getConsumers().getConsumerCount() > 0) {
-      newSummary.addConsumers(IntStream.range(0, message.getConsumers().getConsumerCount())
-          .mapToObj(i -> message.getConsumers().getConsumer(i))
-          .collect(Collectors.toList()));
+      for (int i = 0; i < message.getConsumers().getConsumerCount(); i++) {
```

Comment:
Can we use a local variable here instead of `message.getConsumers().getConsumerCount()` two method calls per loop iteration ?

Reply:
done